### PR TITLE
Initial README draft

### PR DESCRIPTION
**Pull request by TerrenceJChan**
Hey @foxfriends, I drafted up a quick description of what we're up to. Is this alright, or do you have any suggestions on what to add or change?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
-# discord-rpg-experiment
-An experimental Discord bot that simulates RPG-like battles.
+# Discord Adventure RPG (Name TBD)
+
+Slayer of beasts, guardian of the last bastion of humanity, venture to the den of monsters and defeat the enemies of humankind!
+
+This is a text-based adventure RPG played on Discord, a free VoIP app. Individuals and groups of players craft and customize their characters to face-off against powerful enemies in turn-based combat.
```
Comment (foxfriends) [inline README.md]: Maybe make a link to Discord here.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,60 @@
...
+
+Combat is turn-based, with various mechanics including cooldowns and skill variety (think **Pokémon!**) on both the players' and enemy's part, creating a situation where fights should be pre-planned and tactical decisions being made during a fight.
+
+Once a monster is slain, the participating players gain monster-specific crafting material in which they use to craft new gear, be it armours or weapons.
+
+With new gear worn and in hand, players venture off to take on another battle!
+
+## Stacks Used
+
+This game is being developed using **Node.js** and **PostgreSQL**. The game is played over **Discord**, and it takes advantage of the npm package `discord.js` to allow the game's bot to describe the game world and interact with players.
```
Comment (foxfriends) [inline README.md]: Can link to these too

File: README.md [hunk 3]
```diff
@@ -1,2 +1,60 @@
...
+
+## Roadmap
+
+| Alpha (Current)                                              | Closed-Beta (ETA Mid-Late July)                              | Open-Beta (TBD)                                              | Launch                                                       |
+| :----------------------------------------------------------- | :----------------------------------------------------------- | :----------------------------------------------------------- | ------------------------------------------------------------ |
+| Crafting (recipes, item stats, inventory)<br /><br />Fighting monsters (cooldowns, skills)<br /><br />Discord game user account creation<br /><br />Uploading and retrieving player information onto database | Closed Discord server for initial player testing<br /><br />Party features (group combat)<br /><br />Item enhancement and upgrading<br /><br />Initial power-curving design | Open Discord server for larger-scale testing<br /><br />Enemy AI for more engaging fights<br /><br />Power-curving refinement and general balancing | Game Bot available for users to add to their servers<br /><br />Playing the game ourselves! |
```
Comment (foxfriends) [inline README.md:60]: Roadmap usually done with checklist, then you can see exactly which things are done. Easier to write than a mega table anyway, just won't be horizontal. Up to you though

File: README.md [hunk 4]
```diff
@@ -1,2 +1,60 @@
...
+
+Players control an adventuring company of customizable characters. They venture off alone or with a party of other players to fight monsters.
+
+Combat is turn-based, with various mechanics including cooldowns and skill variety (think **Pokémon!**) on both the players' and enemy's part, creating a situation where fights should be pre-planned and tactical decisions being made during a fight.
+
+Once a monster is slain, the participating players gain monster-specific crafting material in which they use to craft new gear, be it armours or weapons.
+
+With new gear worn and in hand, players venture off to take on another battle!
+
+## Stacks Used
```
Comment (foxfriends) [inline README.md]:
```suggestion
## Stack Used
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,60 @@
...
+
+This game is being developed using **Node.js** and **PostgreSQL**. The game is played over **Discord**, and it takes advantage of the npm package `discord.js` to allow the game's bot to describe the game world and interact with players.
+
```
Comment (foxfriends) [inline README.md:44]: Maybe add a section here on "how to set yourself up for development", linking to things you need to download and describing what files to create and how to initialize database and stuff like that

**Review by foxfriends — APPROVED** [review]
Wow most detailed read me I ever saw. A few little things you could add, but Readme content is all up to you

File: README.md (hunk 3 above)
Comment (TerrenceJChan) [reply README.md:60]: Good call, I'll add it for the next iteration!

File: README.md (hunk 5 above)
Comment (TerrenceJChan) [reply README.md:44]: Sounds like a next iteration thing, but aye that's a good call.

**Pull request merged**